container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend_api_container_functions.go

```diff
@@ -466,7 +467,16 @@ func (backend *DockerKurtosisBackend) getMatchingApiContainers(ctx context.Conte
 			}
 		}
 
-		allMatchingApiContainers[containerId] = apicObj
+		bridgeNetworkIpAddress, err := backend.dockerManager.GetContainerIP(ctx, backend.dockerManager.GetBridgeNetworkName(), containerId)
```

Comment:
we already have the bridge network IpAddress from earlier; why do we need to get it again?